client/lib/domains/dns/index.js

```diff
@@ -126,7 +130,49 @@ function getFieldWithDot( field ) {
 	return ( typeof field === 'string' && field.match( /^([a-z0-9-_]+\.)+\.?[a-z]+$/i ) ) ? field + '.' : field;
 }
 
+function isWpcomRecord( record ) {
+	return /^wpcom:/.test( record.id );
+}
+
+function filterRootARecords( domain, records ) {
+	return filter( records, {
+		type: 'A',
+		name: domain + '.'
```

Comment:
You could use a template string here similarly to what you did in `addMissingWpcomRecords()`.